### PR TITLE
Link to sosmeta

### PR DESCRIPTION
**Pull request by WiljamiT**
Added link to sosmeta schemas.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "essote-management-portal",
```
Comment (villejuutila) [inline package-lock.json:8]: Revert this, since property **name** is already defined in package-lock.json row 1.

**Review by villejuutila — CHANGES_REQUESTED** [review]
Comment on package-lock.json left and also why has submodule changed? Revert that aswell.

File: package-lock.json (hunk 1 above)
Comment (WiljamiT) [reply package-lock.json]: Done.

File: src/components/screens/editor/editor-screen-drawer.tsx [hunk 2]
```diff
@@ -187,6 +187,10 @@ const EditorScreenDrawer: FC<Props> = ({
             <FormControlLabel value={ true } control={ <Radio/> } label={ strings.editorScreen.drawer.formTemplateSosmeta }/>
             <FormHelperText>
               { strings.editorScreen.drawer.formTemplateSosmetaHelper }
+              <Link href="https://sosmeta.thl.fi/document-definitions/list" target="_blank">
+                { strings.editorScreen.drawer.formTemplateSosmetaLink }
+              </Link>
+              { strings.editorScreen.drawer.formTemplateSosmetaDot }
```
Comment (EVP1989) [inline src/components/screens/editor/editor-screen-drawer.tsx:193]: I think this can be removed after all, after colon, it's not really needed.

**Review by EVP1989 — APPROVED** [review]
Approved with suggestion

File: src/components/screens/editor/editor-screen-drawer.tsx (hunk 2 above)
Comment (WiljamiT) [reply src/components/screens/editor/editor-screen-drawer.tsx]: done.

**Review by villejuutila — APPROVED** [review]

**Pull request merged**